internal/livelint/check_can_visit_public_app_internal_test.go

```diff
@@ -79,7 +80,7 @@ func TestCheckCanVisitPublicApp(t *testing.T) {
 	cases := []struct {
 		it              string
 		ingresses       []netv1.Ingress
-		roundTripFunc   RoundTripFunc
+		roundTripFunc   livelint.RoundTripFunc
```

Comment:
I think, `RounTripFunc` is only used for testing. Shouldn't it be defined in the `*_test` package?